hugo/content/downloads.md

```diff
@@ -120,7 +120,7 @@ We do not currently provide an official Android client. However, you can use one
 Development snapshots contain unreleased features and changes that will eventually be available in the next stable release. Please report any problems you encounter on [our issue tracker](https://github.com/mumble-voip/mumble/issues).
 
 The most recent snapshot version is the first snapshot of Mumble version 1.4.0
-([Release Announcement](https://www.mumble.info/blog/first-mumble-1.4.0-development-snapshot/)).
+([Release Announcement](https://www.mumble.info/news/first-mumble-1.4.0-development-snapshot/)).
```

Comment:
This is unrelated to this change, but this link should be relative, not absolute; at the very least in source.

Reply:
You mean these `relref` tags?
While I agree that relative linking is better, I also think that a simple syntax is very good.
So if it is possible it should work with something like: `(news/first-mumble-1.4.0-development-snapshot/)`

Reply:
That’s outside our control. We will have to use the hugo mechanism, which is the rel functions. We should not introduce an additional syntax or mechnism, which would wrap that mechanism. The ref syntax is the _simple syntax_. (The alternative would be introducing a shortcode or functions.)

Reply:
Found this: https://github.com/13rac1/goldmark-wikilink

Sadly upstream hugo does not seem to implement it by themselves: https://github.com/gohugoio/hugo/issues/3606

Reply:
I fixed this in master. Unfortunately this will result in a conflict on this branch/PR. But given that it’s a small change, and this PR still has open points, I decided to go ahead and fix it.